app/controllers/coronavirus_form/product_details_controller.rb

```diff
@@ -99,8 +99,8 @@ def sanitized_product(params)
   end
 
   def previous_path
-    return check_your_answers_path if session["check_answers_seen"]
+    return check_your_answers_url if session["check_answers_seen"]
```

Comment:
This check threw me a bit, because it's not on the other pages, but I guess it's because this is a looping question?

Reply:
Yeah I think it's intentional.